### PR TITLE
Base Credential model 

### PR DESCRIPTION
**Pull request by maxalmazov**

### CODE REVIEW

File: models/staging/tpdm/base/base_tpdm__credentials.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        {{ extract_descriptor('v:gradeLevels::string') }}                             as grade_levels,
+        {{ extract_descriptor('v:stateOfIssueStateAbbreviationDescriptor::string') }} as state_of_issue_state_abbreviation,
+        {{ extract_descriptor('v:teachingCredentialDescriptor::string') }}            as teaching_credential,
+        {{ extract_descriptor('v:teachingCredentialBasisDescriptor::string') }}       as teaching_credential_basis,
+        {{ extract_descriptor('v:academicSubjects::string') }}                        as academic_subjects,
+        {{ extract_descriptor('v:certificationRouteDescriptor::string') }}            as certification_route,
+        {{ extract_descriptor('v:credentialStatusDescriptor::string') }}              as credential_status,
+        {{ extract_descriptor('v:educatorRoleDescriptor::string') }}                  as educator_role,
+        -- references
+        v:studentAcademicRecords as student_academic_records,,
```
Comment (zaharenka) [inline models/staging/tpdm/base/base_tpdm__credentials.sql]:
```suggestion
        v:studentAcademicRecords as student_academic_records
```
 Noticed extra commas. Although the rest looks good

File: models/staging/tpdm/base/base_tpdm__credentials.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    from candidates
+)
+select * from renamed
```
Comment (ChristianMurphy) [inline models/staging/tpdm/base/base_tpdm__credentials.sql:41]: Could we add a newline between CTEs?
```suggestion
)

select * from renamed
```

Comment (maxalmazov) [reply models/staging/tpdm/base/base_tpdm__credentials.sql:41]: Hi @ChristianMurphy, it is good idea it looks better with new line, but there is not new line in other models in current repository, if we add new line we break general code style. If it is ok, we will add new line

Comment (ChristianMurphy) [reply models/staging/tpdm/base/base_tpdm__credentials.sql:41]: Both styles are in the existing code

https://github.com/Unicon/edu_edfi_source/blob/c865d2839664e34670a6d73148b7ca9a1b21babe/models/staging/edfi_3/intermediate/int_ef3__deduped_descriptors.sql#L1-L29

File: models/staging/tpdm/base/base_tpdm__credentials.sql [hunk 3]
```diff
@@ -0,0 +1,43 @@
+with candidates as (
+    {{ source_edfi3('credentials') }}
+),
+
```
Comment (holdorph) [inline models/staging/tpdm/base/base_tpdm__credentials.sql]: remove

**Review by holdorph — APPROVED** [review]

**Pull request merged**